workflow-v2.2.0-e2e/README.md

```diff
@@ -0,0 +1,12 @@
+# Workflow 2.0.0 e2e Tests (parallel)
+
+End-to-end tests for the Deis Workflow open source PaaS, executed in parallel.
+
+NOTE: This assumes a fresh `helmc install` of Deis v2. The test suite creates
+an initial user with admin privileges (who is deleted when tests complete).
+See https://github.com/deis/workflow-e2e for more detail.
+
+```console
+$ helmc uninstall -n deis -y workflow-e2e ; helmc install workflow-e2e
```

Comment:
All three instances of `workflow-e2e` here need to be changed to `workflow-v2.2.0-e2e`. They won't run correctly otherwise.